aixplain/factories/agent_factory/__init__.py

```diff
@@ -44,7 +44,7 @@ class AgentFactory:
     def create(
         cls,
         name: Text,
-        llm_id: Text,
+        llm_id: Text = "669a63646eb56306647e1091",
```

Comment:
Please add `Optional` in the method definition for all optional parameters e.g. `Optional[llm_id]`.

Reply:
I think the use of `Optional` in type annotations in Python is intended to indicate that a value can either be of the specified type or `None`, no?. Since it has a not null value there is not need to use Optional. Please confirm.

Reply:
yes, you're correct. Didn't know that actually: https://docs.python.org/3/library/typing.html#typing.Optional